app/views/users/show.html.erb

```diff
@@ -6,6 +6,7 @@
         <li class="price"><%= @user.first_name %> <%= @user.last_name %></li>
         <li class="bullet-item"><%= @user.email %></li>
         <li class="bullet-item"><%= avatar(300) %></li>
+        <li class="bullet-item"><%= image_tag(@user.avatar.url, alt: @user.avatar.url) %></li>
```

Comment:
the avatar url really doesn't really add value to someone that is visually impaired. Consider the alternative alt text of "#{@user.name} Avatar"